src/forms/toggle/Toggle.tsx

```diff
@@ -21,6 +21,7 @@ type ToggleProps = {
     checked?: boolean;
     defaultChecked?: boolean;
     onChange?: (newValue: boolean) => void;
+    onClick?: (evt: React.MouseEvent<HTMLInputElement>) => void;
```

Comment:
Since it is toggle component I guess onChange is sufficient. is there any specific case where you need onClick ?

Reply:
Please, add default props for undefined function. It can be just () => {}

Reply:
Done